### ISSUE TITLE
Its not open

### ISSUE THREAD

**Issue by Gloster10** [mfrasca/xskat#9]
Why it is on Github  ?
It is not open source.

**Comment by maweki** [comment]
What makes you think that? The original license is quite clear:

*  This program is free software; you can redistribute it freely.
* Redistribution of modified versions is permitted

**Comment by Gloster10** [comment]
@maweki, please show me the source code.

**Comment by maweki** [comment]
Right here???

https://github.com/mfrasca/xskat/blob/master/skat.c

**Comment by Gloster10** [comment]
Oh, I see, before and after my hint, the source zip : you can get here : https://we.tl/t-TvS6udPQDD
Further comments maweki ?

**Comment by maweki** [comment]
I don't understand what you're on about. The source code is here on github. The license is also fine.

**Comment by Gloster10** [comment]
You have both zip sources, you did compare ?

**Comment by mfrasca** [comment]
hi. I'm not sure I see the problem. when I created this repository, I collected as many versions (zip files) of the software I could find, ordered them chronologically, and converted the sequence of zip files into a git repository, which I published here.
the only difference is with the original cards deck, which I had to remove from history, because the copyright owner did not agree with keeping it here, even at the ludicrously low resolution this program uses.
the licence is not as "free" as one would wish, but it is clearly open, where does @Gloster10 see the problem?
it would however be very nice if the copyright owner of this program would chime in, and choose a clearer and freer licence. this custom-written thing is, in my opinion, not easy to respect literally.

**Comment by Gloster10** [comment]
@mfrasca, please refer to the link :  https://we.tl/t-TvS6udPQDD, you will find total different zip-files. 
Before and after I did open the issue.

**Comment by mfrasca** [comment]
I followed the link, it requires me to accept some terms of use, I click on "I don't agree", and I don't see what you mean. should you want to reopen this issue, please provide some self-contained, self-explicative material, in whatever for you wish to provide it, but let's stay here on github.

**Comment by Gloster10** [comment]
@mfrasca : "self-contained, self-explicative material", I  am sorry, I did get it from githhub, what else ?

**Comment by mfrasca** [comment]
I mean that "please refer to the link" is the opposite of self-contained.
Can you please write here in this issue (which you opened and subsequently
closed) what is the problem, that you described in that we-transfer zip file?

**Comment by Gloster10** [comment]
I am sorry, you did share it.  Therefore  please explain me the difference, what given, before my Isue, and afterwards., of the source code, in the linked zip-fiiles. I am sure, github will have the history of your files.

**Comment by mfrasca** [comment]
please show whatever you intend to show here in this repository. maybe I didn't understand what you're writing, but if there's any difference with a zip file you grabbed somewhere else, &lt;sorry-if-rude-but> what do we care about it?</>
(I'm unsubscribing until there's some workable information)

**Comment by Gloster10** [comment]
Sorrry to say, but mfrasca is a liar !

**Comment by maweki** [comment]
> Sorrry to say, but mfrasca is a liar !

What the heck are you on about? You can clone this repository, `configure` and `make` it and then execute the `xskat` application. If this repository does not contain the `xskat` source code, what does it contain then?

Can you explain - in plain english - what your wetransfer link contains and how it would have any bearing on this github project and the code it contains?

**Comment by maweki** [comment]
Part of your - let's call it "confusion" - is the following: the source you uploaded to wetransfer is from xskat for android written in java (for example, from here https://github.com/rnauber/xskat-android).

This repository here is the original xskat in C.

So they are different applications for different platforms (same original author and license though).

Otherwise, I am still not sure what exactly you're droning on about.

**Comment by mfrasca** [comment]
hi @maweki thank you for participating to this otherwise confusing thread. 
after them calling me names, I have blocked the other account, and reported them.

as I stated above, I only created this repository because I thought it would be a service to the community: I found several archives of this software, and the license was free enough to let me publish them here all together as a single git repository.

it's a pity the original author did not choose for a truly free license, where we could continue working at this program under the same name. this is just repackaging.

thanks also for linking to the Android version of xskat, I wasn't aware of it.
